lib/Person.php

```diff
@@ -10,12 +10,12 @@
  * @property string $id
  * @property string $object
  * @property string $account
- * @property mixed $address
- * @property mixed $address_kana
- * @property mixed $address_kanji
+ * @property \Stripe\StripeObject $address
+ * @property \Stripe\StripeObject|null $address_kana
+ * @property \Stripe\StripeObject|null $address_kanji
  * @property int $created
  * @property bool $deleted
- * @property mixed $dob
+ * @property \Stripe\StripeObject $dob
```

Comment:
can't this be null? IF not is it a weird hash with January 1 1970?

Reply:
Not according to the spec, but I think the spec may be wrong in this instance.